### PR TITLE
Add deprecation warning for statsd output

### PR DESCRIPTION
**Pull request by heitortsergent**
Related to https://github.com/grafana/k6/pull/3347, closes https://github.com/grafana/k6-docs/issues/1354.

### CODE REVIEW

**Comment by heitortsergent** [comment]
@mstoykov I did a quick search on the docs for this, and it looks like the [Amazon CloudWatch](https://k6.io/docs/results-output/real-time/amazon-cloudwatch/), [Datadog](https://k6.io/docs/results-output/real-time/datadog/), [Netdata](https://k6.io/docs/results-output/real-time/netdata/), and [New Relic](https://k6.io/docs/results-output/real-time/new-relic/) integration pages all had instructions that included the `--out statsd` command.

Should I update their instructions so it includes instructions for how to build k6 with the new statsd extension, similar to the [Apache Kafka](https://k6.io/docs/results-output/real-time/apache-kafka/) page?

**Comment by mstoykov** [comment]
Hi @heitortsergent, this is a good catch and yes - let's do that.

**Comment by heitortsergent** [comment]
@mstoykov made some updates to all the pages that had the `--out statsd` output, let me know what you think. :)

File: src/data/markdown/translated-guides/es/01 Get started/04 Results Output.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ k6 puede enviar datos de resultados más granulares a diferentes salidas para in
 <CodeGroup labels={[]}>
 
 ```bash
-$ k6 run --out statsd script.js
+$ k6 run --out output-statsd script.js
```
[the remark this answers is not in the document]
Comment (heitortsergent) [reply src/data/markdown/translated-guides/es/01 Get started/04 Results Output.md:106]: Ah good catch @codebien. It looks like there are some differences between the English and Spanish version for this page... I'm going to update the code snippet so they match and it doesn't use the `output-statsd` option.

File: src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Netdata.md [hunk 2]
```diff
@@ -24,9 +37,11 @@ Netdata runs a fully functional [StatsD](https://learn.netdata.cloud/docs/agent/
...
 
 ```bash
-k6 run --out statsd script.js
+k6 run --out output-statsd script.js
 ```
 
+Make sure you're using the k6 binary you built with the xk6-output-statsd extension.
+
 **Caveat**: By default, Netdata binds the StatsD server to `localhost`. That means that if Netdata and k6 are in different hosts, you will need to edit the configuration file of Netdata.
 1. Visit [StatsD documentation](https://learn.netdata.cloud/docs/agent/collectors/statsd.plugin) for a reference on the configuration options. We are interested in `#bind to` option.
```
[the remark this answers is not in the document]
Comment (heitortsergent) [reply src/data/markdown/translated-guides/en/04 Results output/200 Real-time/00 Netdata.md]: Fixed them! I replaced them with the [GitHub link](https://github.com/netdata/netdata/blob/master/collectors/statsd.plugin/README.md) @yorugac mentioned in our Slack.

**Pull request merged**